modules/machineset-yaml-rhv.adoc

```diff
@@ -0,0 +1,110 @@
+{rh-virtualization} and creates nodes that are labeled with
+`node-role.kubernetes.io/<node_role>: ""`
+
+In this sample, `<infrastructure_id>` is the infrastructure ID label that is based on the cluster ID that you set when you provisioned the cluster, and `<role>` is the node label to add.
+
+[source,yaml,subs="+quotes"]
+----
+    apiVersion: machine.openshift.io/v1beta1
+    kind: MachineSet
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+        machine.openshift.io/cluster-api-machine-role: <role> <2>
+        machine.openshift.io/cluster-api-machine-type: <role> <2>
+      name: <infrastructure_id>-<role> <3>
+      namespace: openshift-machine-api
+    spec:
+      replicas: <number_of_replicas> <4>
+      Selector: <5>
+        matchLabels:
+          machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+          machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+      template:
+        metadata:
+          labels:
+            machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+            machine.openshift.io/cluster-api-machine-role: <role> <2>
+            machine.openshift.io/cluster-api-machine-type: <role> <2>
+            machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+        spec:
+          metadata:
+            labels:
+              node-role.kubernetes.io/<role>: "" <2>
+          providerSpec:
+            value:
+              apiVersion: ovirtproviderconfig.machine.openshift.io/v1beta1
+              cluster_id: <ovirt-cluster-id> <6>
+              template_name: <ovirt-template-name> <7>
+              instance_type_id: <instance-type-id> <8>
+              cpu: <9>
+                cores: <number-of-cores> <10>
+                sockets: <number-of-sockets> <11>
+                threads: <number-of-threads> <12>
+              memory_mb: <memory-size> <13>
+              os_disk: <14>
+                size_gb: <disk-size> <15>
+              network_interfaces: <16>
+                vnic_profile_id:  <vnic_profile_id> <17>
+              credentialsSecret:
+                name: ovirt-credentials <18>
+              kind: OvirtMachineProviderSpec
+              type: <workload-type> <19>
+              userDataSecret:
+                name: worker-user-data
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI (oc) installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+<2> Specify the node label to add.
+
+<3> Specify the infrastructure ID and node label.
+
+<4> Specify the number of machines to create.
+
+<5> Selector for the machines.
+
+<6> Specify a UUID for the oVirt cluster to which this VM instance belongs.
```

Comment:
Just checking - is the UUID actually something that the user specifies? And if so, should "a" be "the"?

Reply:
The user does specify it, according to the SME. And yes, it should be "the".